mindmeld/models/taggers/pytorch_crf.py

```diff
@@ -0,0 +1,408 @@
+import logging
+import os
+import random
+import uuid
+from collections import Counter
+from copy import copy
+from itertools import zip_longest
+from random import randint
+
+import numpy as np
+import torch
```

Comment:
Note for future: Since we plan to include torch as a requirement in MindMeld, some snippet in neural network codebase can be modified to directly `import torch` and use `nn.Module` and other torch functions directly.

Reply:
Raised issue https://github.com/cisco/mindmeld/issues/415 for this!